commands/csr-slash.js

```diff
@@ -0,0 +1,63 @@
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+const getCSR = require('../utils/getCSR');
+const buildRankEmbeds = require('../utils/buildRankEmbeds');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('csr')
+		.setDescription("Get the CSR for a user by entering their gamertag.")
+		.addStringOption((option) =>
+		option
+		  .setName("gamertag")
+		  .setDescription("A valid Xbox Live gamertag")
+		  .setMaxLength(15)
+		  .setRequired(true)
+	  	),
+	async execute(interaction) {
+		try {
+			// console.log(`csr-slash.js - About to set const: gamertag`)
```

Comment:
Generally we want to wipe out any commented debug code before merging things in.